lib/ignored-paths.js

```diff
@@ -146,6 +163,31 @@ class IgnoredPaths {
                 this.baseDir = path.dirname(path.resolve(options.cwd, ignorePath));
                 this.addIgnoreFile(this.ig.custom, ignorePath);
                 this.addIgnoreFile(this.ig.default, ignorePath);
+            } else {
+                try {
+
+                    // if the ignoreFile does not exist, check package.json for eslintIgnore
+                    const packageJSONPath = findPackageJSONFile(options.cwd);
+
+                    if (packageJSONPath) {
+                        let packageJSONOptions;
+
+                        try {
+                            packageJSONOptions = JSON.parse(fs.readFileSync(packageJSONPath, "utf8"));
+                        } catch (e) {
+                            debug("Could not read package.json file to check eslintIgnore property");
+                        }
+
+                        if (packageJSONOptions.eslintIgnore && Array.isArray(packageJSONOptions.eslintIgnore)) {
```

Comment:
makes sense @kaicataldo 

updated to throw an error when it isn't an array and added a test for that. I didn't write error tests for when the package.json isn't found or when package.json is in a bad format, as they don't relate to the behavior of the eslintIgnore property. They just throw, which will give a decent error message.